src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1829,6 +1958,13 @@ def create_privacy_request_func(
                     },
                 )
                 queue_privacy_request(privacy_request.id)
+            else:
+                pre_approval_webhooks = db.query(PreApprovalWebhook).all()
+                for webhook in pre_approval_webhooks:
+                    privacy_request.trigger_pre_approval_webhook(
+                        webhook=webhook,
+                        policy_action=privacy_request.policy.get_action_type(),
+                    )
```

Comment:
Let's see if we can share this whole conditional block with the similar code in the other endpoint!